source/plot/SvgWriter2D.ooc

```diff
@@ -43,13 +41,12 @@ SvgWriter2D: class {
 	init: func ~withPositioningFilename (filename: String, =numberOfPlotsHorizontally, args: ...) {
 		this init(File new(filename), args)
 	}
-	init: func ~svgPlots (file: File, svgPlots: VectorList<SvgPlot>) {
-		this init(file)
-		this svgPlots = svgPlots
```

Comment:
This assigned new object to `this svgPlots` without deleting old one (same below).